feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt

```diff
@@ -0,0 +1,116 @@
+package com.iteneum.notification.presentation
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.notification.R
+import com.iteneum.notification.domain.models.NotificationDomain
+
+@Composable
+fun NotificationListView() {
```

Comment:
Please add the documentation to this function

Reply:
I added comments